SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -21,58 +21,59 @@ is_future_reef_map_file() {
     echo $file | egrep -q '^FutureReefMap_'
 }
 
+# notify_recipients
+# notify uploader and backup recipient about status pof uploaded file
+# $1 - file name
+# $2 - message
+notify_recipients() {
+    local file=$1; shift
+    local message=$1; shift
+    local recipient=`get_uploader_email $file`
+
+    if [ -n "$recipient" ]; then
+        echo "" | notify_by_email $recipient $message
+    fi
+    echo "" | notify_by_email $BACKUP_RECIPIENT $message
+}
+
 # handles a single netcdf file, return path in which file is stored
 # $1 - netcdf file
 handle_netcdf_file() {
     local file=$1; shift
+
     log_info "Handling SOOP CO2 file '$file'"
 
-    local recipient=`get_uploader_email $file`
-    if [ -n "$recipient" ]; then
-        echo "" | notify_by_email $recipient "Processing new underway CO2 file '$file'"
-    fi
-    echo "" | notify_by_email $BACKUP_RECIPIENT "Processing new underway CO2 file '$file'"
+    echo "" | notify_by_email $BACKUP_RECIPIENT  "Processing new underway CO2 file '$file'"
 
-    local tmp_file=`make_writable_copy $file`
     local tmp_file_with_sig
-    local checks='cf imos'
+    local checks='cf imos:1.3'
 
-    if is_imos_soop_co2_file $tmp_file; then
-        tmp_file_with_sig=`trigger_checkers_and_add_signature $tmp_file $BACKUP_RECIPIENT $checks`
-    elif is_future_reef_map_file $tmp_file; then
+    if is_imos_soop_co2_file $file; then
+        tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
+    elif is_future_reef_map_file $file; then
         checks='cf'
-        tmp_file_with_sig=`trigger_checkers_and_add_signature $tmp_file $BACKUP_RECIPIENT $checks`
+        tmp_file_with_sig=`trigger_checkers_and_add_signature $file $BACKUP_RECIPIENT $checks`
     else
-        file_error "Not an underway CO2 file '$file'"
-        rm -f  $tmp_file_with_sig $tmp_file
+        file_error_and_report_to_uploader $BACKUP_RECIPIENT "Not an underway CO2 file '$file'"
```

Comment:
Could shorten the message a bit by using `basename $file` here. The path won't mean anything to the uploader anyway.